Source/WebKit/Scripts/webkit/tests/SerializedTypeInfo.cpp

```diff
@@ -100,6 +101,8 @@ Vector<SerializedTypeInfo> allSerializedTypes()
             "float"_s,
             "float"_s,
         } },
+        { "Ref<WebCore::TimingFunction>"_s, {
```

Comment:
@achristensen07 does this look correct?

Reply:
No.  This information will need to be exposed to the IPC testing API in a new way that doesn't use allSerializedTypes.  Let's omit that for now.